src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -252,77 +295,78 @@ s32 func_80B142F4(EnTa* this, PlayState* play, u16 textId) {
 
     if ((ABS((s16)(this->actor.yawTowardsPlayer - this->actor.shape.rot.y)) <= 0x4300) &&
         (this->actor.xzDistToPlayer < 100.0f)) {
-        this->unk_2E0 |= 1;
+        this->stateFlags |= TALON_STATE_FLAG_TRACKING_PLAYER;
         func_8002F2CC(&this->actor, play, 100.0f);
     }
     return false;
 }
 
-void func_80B14398(EnTa* this, PlayState* play) {
+void EnTa_SleepTalkInKakariko(EnTa* this, PlayState* play) {
     if (Actor_TextboxIsClosing(&this->actor, play)) {
-        func_80B13AA0(this, func_80B14754, func_80B167FC);
+        EnTa_SetupAction(this, EnTa_IdleAsleepInKakariko, EnTa_AnimSleeping);
     }
 }
 
-void func_80B143D4(EnTa* this, PlayState* play) {
+void EnTa_SleepTalkInLonLonHouse(EnTa* this, PlayState* play) {
     if (Actor_TextboxIsClosing(&this->actor, play)) {
-        func_80B13AA0(this, func_80B146F8, func_80B167FC);
+        EnTa_SetupAction(this, EnTa_IdleAsleepInLonLonHouse, EnTa_AnimSleeping);
     }
 }
 
-void func_80B14410(EnTa* this) {
+void EnTa_SetupAwake(EnTa* this) {
     if (!LINK_IS_ADULT) {
-        func_80B13AA0(this, func_80B14C18, func_80B167C0);
-        SET_EVENTCHKINF(EVENTCHKINF_13);
+        EnTa_SetupAction(this, EnTa_IdleAwakeInCastle, EnTa_AnimRepeatCurrent);
+        SET_EVENTCHKINF(EVENTCHKINF_TALON_WOKEN_IN_CASTLE);
     } else {
-        func_80B13AA0(this, func_80B14CAC, func_80B167C0);
-        SET_EVENTCHKINF(EVENTCHKINF_6A);
+        EnTa_SetupAction(this, EnTa_IdleAwakeInKakariko, EnTa_AnimRepeatCurrent);
+        SET_EVENTCHKINF(EVENTCHKINF_TALON_WOKEN_IN_KAKARIKO);
     }
 }
 
-void func_80B1448C(EnTa* this, PlayState* play) {
+void EnTa_TalkAfterAwakeUp(EnTa* this, PlayState* play) {
```

Comment:
"awake up" :sweat_smile:

Reply:
lmfao 
maybe it can be TalkAwake, as opposed to trying to phrase it "talk after doing something"?

Reply:
Great search-replace here... 😅 

Naming this is a bit tricky: this is just an intermediate state in the wake-up sequence, so TalkAwake would be confusing with the "fully awake" states already named `TalkAwakeIn.*` imo. The sequence goes something like this:

- `EnTa_WakeUp`: plays wake up animation, sets up `EnTa_TalkWakingUp`
- `EnTa_TalkWakingUp`: Talking eyes closed, but already standing up. Sets up `EnTa_TalkAfterWokenUp`
- `EnTa_TalkAfterWokenUp`: Talking standing up, eyes half-open. Sets up the "fully awake" actions (`EnTa_SetupAwake`)

The previous name `EnTa_TalkAfterWokenUp` was not great either. `TalkWakingUp2`? `TalkAlmostAwake`? 😄

Reply:
I have no idea
after a lot of time thinking about it, huh...
`EnTa_TalkWakingUp` -> `EnTa_TalkWokenUp`
`EnTa_TalkAfterWokenUp` -> `EnTa_TalkWakingUp`
:shrug:

Reply:
tbh WakingUp1 and WakingUp2 kinda sounds better to me than trying to give them unique descriptions. it sounds like its part of the same process of waking up to me